international.py

```diff
@@ -28,7 +29,10 @@
             'nl': 'Verwijder selectie',
             'ur': 'انتخاب کو مٹا دیں',
             'sv': 'Rensa urval',
+            'pt': 'Anular Seleção',
+            'lux': 'Auswiel obhiewen'
```

Comment:
comma missing on almost all entries ;-) 

Sorry!

Reply:
Upsi! Changed it, should work now! 
Thanks